src/main/java/qna/domain/Answer.java

```diff
@@ -43,6 +46,17 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
+    public void checkDelete(User loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+    }
+
+    public void delete(List<DeleteHistory> deleteHistories) {
+        setDeleted(true);
```

Comment:
```suggestion
        this.deleted = true;
```

굳이 set 메서드로 구현해야 하나?